src/app/webapp/user/signout/constants.tsx

```diff
@@ -43,6 +44,12 @@ const settings = [
     url: "https://github.com/PROJECT-PIPLUP/lounas/issues/new/choose",
     isInternal: false,
   },
+  {
+    title: "アプリ開発サークル",
+    icon: <Twitter /> /** @todo Replace */,
```

Comment:
@Shvain 

後は、アプリ開発サークルのSVGアイコンのみ
`/public/icons/`に入れて、`import`後に以下を書き換えればOK

```suggestion
    icon: <Icon /> /** ここは`import`した名前 */,
```

Reply:
修正確認クローズ